pureskillgg_csgo_dsdk/scrubber/scrub_pii.py

```diff
@@ -0,0 +1,162 @@
+"""
+Scrub PII from your data
+"""
+# pylint: disable=invalid-name
+# pylint: disable=c-extension-no-member
+
+from typing import Dict, Union
+from numbers import Number
+import rapidjson
+
+REDACTED = "redacted"
+
+SCRUB_CSDS_PII_CHANNEL_INSTRUCTIONS = [
+    {"channel": "player_name"},
+    {"channel": "header"},
+    {"channel": "player_personal"},
+    {"channel": "player_info"},
+    {"channel": "player_status"},
+]
+
+
+def scrub_csds_pii(manifest: Dict, data: Dict):
+    """Remove events in overtime rounds
+
+    Inputs:
+        manifest (dict): CSDS manifest.
+        data (dict): CSDS data dictionary.
+
+    Returns:
+        Nothing. Data is changed in place.
```

Comment:
Update this to document we returned the new manifest.

Reply:
Done